layer/cache_sideload/cache_sideload_layer.cc

```diff
@@ -46,22 +51,26 @@ class CreateCacheEvent : public Event {
       : Event(name),
         path_("path_", path),
         data_size_("initial_data_size", data_size),
-        cache_size_("cache_size", cache_size) {
-    InitAttributes({&path_, &data_size_, &cache_size_});
+        cache_size_("cache_size", cache_size),
+        trace_attr_("trace_attr", "cache_sideload_layer", "i",
+                    {&scope_, &path_, &data_size_, &cache_size_}) {
+    InitAttributes({&path_, &data_size_, &cache_size_, &trace_attr_});
   }
 
  private:
   StringAttr path_;
   Int64Attr data_size_;
   Int64Attr cache_size_;
+  StringAttr scope_ = StringAttr("scope", "t");
```

Comment:
You can initialize it directly: `Type x(a, b)`

Reply:
I think I need to use curly brackets instead of parentheses.
https://stackoverflow.com/questions/24836526/why-cant-member-initializers-use-parentheses